Formula/docker-ls.rb

```diff
@@ -0,0 +1,36 @@
+class DockerLs < Formula
+  desc "Tools for browsing and manipulating docker registries."
+  homepage "https://github.com/mayflower/docker-ls"
+  url "https://github.com/mayflower/docker-ls.git",
+    :using => :git,
```

Comment:
This is redundant.

Reply:
The `:using => :git`, I presume? I'll remove it.